tests/h/traversal/roots_test.py

```diff
@@ -312,12 +313,14 @@ def test_it_does_not_assign_create_permission_without_user_role(self, pyramid_co
 
         assert not pyramid_request.has_permission('create', context)
 
-    def test_it_returns_the_group_if_it_exists(self, factories, group_factory):
+    def test_getitem_returns_fetched_group_if_not_None(self, factories, group_factory, group_service):
         group = factories.Group()
+        group_service.fetch.return_value = group
 
         assert group_factory[group.pubid] == group
 
-    def test_it_raises_KeyError_if_the_group_doesnt_exist(self, group_factory):
+    def test_getitem_raises_KeyError_if_fetch_returns_None(self, group_factory, group_service):
+        group_service.fetch.return_value = None
```

Comment:
I'm wondering if we need a test that it calls `fetch()` with the right `pubid` too? `group_service.fetch.assert_called_once_with("pubid")`?